src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationsMigrationService.kt

```diff
@@ -231,7 +222,7 @@ private fun Evidence.toEvidence() = MigrateEvidence(
   },
   details = this.detail,
   reporter = this.createdByUsername,
-  dateAdded = this.date.atStartOfDay().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME),
+  dateAdded = this.date.format(DateTimeFormatter.ISO_LOCAL_DATE),
 )
```

Comment:
I think this must really be a date - swagger is not clear, but naming convention suggest just a date